scalafix-core/src/main/scala/scalafix/patch/Patch.scala

```diff
@@ -110,6 +110,18 @@ object Patch {
   def addLeft(tree: Tree, toAdd: String): Patch =
     tree.tokens.headOption.fold(Patch.empty)(addLeft(_, toAdd))
 
+  def addAround(tok: Token, left: String, right: String): Patch =
+    Add(tok, left, right)
+
+  // if the tree contains more than one token, this patch is not atomic
```

Comment:
That looks like something we want in the Scaladoc
```suggestion
  /** If the tree contains more than one token, this patch is not atomic. */
```